src/odb/src/db/dbBTerm.cpp

```diff
@@ -273,8 +281,11 @@ dbIStream& operator>>(dbIStream& stream, _dbBTerm& bterm)
   stream >> bterm._name;
   stream >> bterm._next_entry;
   stream >> bterm._net;
+  stream >> bterm._mnet;
   stream >> bterm._next_bterm;
   stream >> bterm._prev_bterm;
+  stream >> bterm._next_modnet_bterm;
+  stream >> bterm._prev_modnet_bterm;
```

Comment:
No backward compatibility code - new fields need to be conditional on the schema version